tests/integration/features/istio-jwt/istio_jwt_scopes.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Exposing two endpoints with Istio JWT authorization strategy with scopes
+
+  Scenario: IstioJWTScopes: Calling an endpoint secured with JWT with a valid token
+    Given IstioJWTScopes: There is a deployment secured with JWT on path "/ip"
+    Then IstioJWTScopes: Calling the "/ip" endpoint without a token should result in status between 400 and 403
```

Comment:
The unhappy steps that validate general JWT behaviour are a already tested in IstioJWT feature. I think we should not duplicate them.